crew

```diff
@@ -27,6 +27,9 @@ else
   CREW_NPROC = ENV["CREW_NPROC"]
 end
 
+# Set CREW_NOT_COMPRESS from environment variable
+CREW_NOT_COMPRESS = ! File.exist?("#{CREW_PREFIX}/bin/compressdoc") || ENV["CREW_NOT_COMPRESS"]
```

Comment:
Since when did users know what is best? :-P  Sorry, now I'm sounding like a certain company in the northwest corner of the US.  Compressing manpages by default is actually not a bad idea in this case since Chromebooks have notoriously small hdds.